src/elements/content-sidebar/activity-feed/common/activity-message/__tests__/ActivityMessage.test.js

```diff
@@ -136,4 +136,26 @@ describe('elements/content-sidebar/ActivityFeed/common/activity-message', () =>
         expect(wrapper.state('isTranslation')).toBe(false);
         expect(wrapper.state('isLoading')).toBe(false);
     });
+
+    test('should not show label `(edited)` when `isEdited` prop is false', () => {
+        const comment = {
+            tagged_message: 'Hi ﹫[123:Half] ＠[222:Full] @[432:Latin]',
+            isEdited: false,
+        };
+
+        const wrapper = shallow(<ActivityMessage id="123" {...comment} />);
+
+        expect(wrapper.find('.bcs-ActivityMessage-edited').length).toEqual(0);
```

Comment:
enzyme is used for tests here; I don't think enzyme can check for the user facing text. `data-testid` would be possible because enzyme supports attributes as selectors, but I would rather not do it. `data-testid` leaks to production so I tend to not use it.
With enzyme I can use `FormattedMessage` id prop like so:
```suggestion
        expect(wrapper.exists({ id: 'be.contentSidebar.activityFeed.common.editedMessage' })).toBe(expected);
```
Would that be fine with you? Or should you prefer going with testing-library?

Reply:
We use `data-testid` in many places already today, so it leaking to production isn't a new concern. That said, the message descriptor selector seems fine, too.